packages/svelte2tsx/svelte-jsx.d.ts

```diff
@@ -245,6 +249,24 @@
       // Transition Events
       ontransitionend?: TransitionEventHandler<T>;
       ontransitionendcapture?: TransitionEventHandler<T>;
+
+      // Window Events
+      onafterprint?: GenericWindowEventHandler<T>;
```

Comment:
As mentions here https://github.com/sveltejs/language-tools/pull/388#discussion_r463409452
I think these events don't apply to `HTMLElement`. It should be moved to `SvelteWindowProps`.